script/NumScript.g4

```diff
@@ -115,6 +115,7 @@ valueAwareSource
 statement
   : PRINT expr=expression # Print
   | SET_TX_META '(' key=STRING ',' value=expression ')' # SetTxMeta
+  | SET_ACCOUNT_META '(' acc=ACCOUNT ',' key=STRING ',' value=expression ')' # SetAccountMeta
```

Comment:
Should it be allowed to provide a var for the account? (such as `set_account_meta($foo, "bar", 1)`)
Currently, I think this only allows a literal (`ACCOUNT` is only the token for an account literal)
If we do need support for vars, it should be turned into an expression, and its type checked during compilation from the returned type of `VisitExpr`

Reply:
It totally makes sense thanks! It just added the support for vars. Can you please check it again? :)